__tests__/app.test.js

```diff
@@ -145,7 +145,7 @@ describe("News app",()=>{
         return request(app).patch('/api/articles/Bad-id')
```

Comment:
The test name needs updating to reflect the new message

Reply:
Thanks, will update and merge